controllers/application_controller.go

```diff
@@ -50,16 +54,34 @@ type ApplicationReconciler struct {
 func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
 	app := argocd.Application()
-	if err := r.client.Get(ctx, req.NamespacedName, app); err != nil {
-		return ctrl.Result{}, client.IgnoreNotFound(err)
+
+	err := r.client.Get(ctx, req.NamespacedName, app)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return ctrl.Result{}, err
+	}
+
+	if apierrors.IsNotFound(err) && req.Namespace == r.config.ArgoCD.Namespace {
+		logger.Info("argocd application was deleted")
```

Comment:
It seems good to write the application name.

Reply:
I don't think it is necessary.
In reconcile function, the logger includes `name` and `namespace`.

https://github.com/kubernetes-sigs/controller-runtime/blob/e42a8c17945360813e64bde2c4ff742515b750ba/pkg/builder/controller.go#L325
https://zenn.dev/zoetro/articles/760346baab7e24#controller-runtime%E3%81%8C%E8%A8%AD%E5%AE%9A%E3%81%97%E3%81%A6%E3%81%84%E3%82%8Bkeyvalues